app/_src/kubernetes-ingress-controller/guides/services/custom-entity.md

```diff
@@ -209,8 +213,15 @@ The `curl` command should return
 
 which matches the data inserted in the previous steps.
 
-[hasura]: https://hasura.io/
-<!-- >
-Need to be updated when custom resource reference page is updated.
-[KongCustomEntity]: /reference/custom-resources/
-<-->
+{% if_version gte:3.4.x %}
+
+## Invalid `KongCustomEntity` Configuration
```

Comment:
```suggestion
## Troubleshooting
### Invalid `KongCustomEntity` Configuration
```
Let's nest it under a troubleshooting section, otherwise the heading/section name feels out of place.